doc/news/_preparation_next_release.md

```diff
@@ -75,171 +65,79 @@ As long as it is active, a global lock will be created to ensure no two processe
 This behavior is similar as to when multiple processes will write to the same configuration file.
 Applications should already handle this case gracefully, and just retry writing their configuration.
 
-### New Changetracking API
-
-### <<HIGHLIGHT>>
-
 ### ODBC Backend
 
-Based on the new and more versatile concept for [backends](/doc/dev/backend-plugins.md),
-where backends are implemented as plugins, a new backend-plugin that uses ODBC data sources for storing keys has been developed.
+Based on the new and more versatile concept for [backends](/doc/dev/backend-plugins.md), where backends are implemented as plugins, a new backend-plugin that uses ODBC data sources for storing keys has been developed.
 It was tested on Gentoo Linux with [unixODBC](https://www.unixodbc.org) using [SQLite](https://www.sqlite.org) and [PostgreSQL](https://www.postgresql.org) data sources.
 The ODBC backend-plugin can only be built if the ODBC library is available on the build system. This can be accomplished by installing e.g. unixODBC.
 Microsoft ODBC (on MS Windows) and [iODBC](https://www.iodbc.org) should also be supported, but were not tested yet.
 If you use the plugin with another ODBC implementation as unixODBC, you are very welcome to update the documentation with your experiences!
 
 The [tutorial](../tutorials/odbc-backend.md) is a good place for getting started with the new ODBC backend for Elektra.
 
-### <<HIGHLIGHT>>
-
 ## Plugins
 
 The following text lists news about the [plugins](https://www.libelektra.org/plugins/readme) we updated in this release.
 
-### <<Plugin>>
-
 ### General
 
 - Updated target name of shared object files according to [#3486](https://issues.libelektra.org/3486)
 
 ### spec
 
 - Remove metakeys from array elements correctly [#4961](https://issues.libelektra.org/4961) _(Tomislav Makar @tmakar)_
-- <<TODO>>
-- <<TODO>>
 
 ### recorder
 
 - Add recorder plugin. _(Maximilian Irlinger @atmaxinger)_
-- <<TODO>>
-- <<TODO>>
 
 ### jdbc
 
 - Fix formatting/spelling problem in README.md _(Hannes Laimer @hannes99)_
-- <<TODO>>
-- <<TODO>>
 
 ### backend_odbc
 
-- Add write functionality to the ODBC backend (INSERT, UPDATE, DELETE) _(Florian Lindner @flo91)_
-- <<TODO>>
-- <<TODO>>
-
-### <<Plugin>>
+- Add ODBC backend _(Florian Lindner @flo91)_
 
 ### ansible
 
 - Add `ansible` plugin for exporting keysets as [ansible-libelektra](https://galaxy.ansible.com/elektra_initiative/libelektra) playbooks. _(Maximilian Irlinger @atmaxinger)_
-- <<TODO>>
-- <<TODO>>
-
-### logchange
-
-- <<TODO>>
-- <<TODO>>
-- <<TODO>>
 
 ### toml
 
 - Fix error reporting when unsupported metakey has been encountered. _(Maximilian Irlinger @atmaxinger)_
-- <<TODO>>
-- <<TODO>>
 
 ### Xfconf
 
 - Make Xfconf storage plugin compatible with new backend and noresolver. _(Richard Stöckl @Eiskasten)_
-- <<TODO>>
-- <<TODO>>
-
-### <<Plugin>>
-
-- <<TODO>>
-- <<TODO>>
-- <<TODO>>
+- Implemented the first revision of the Xfconf binding. _(Richard Stöckl @Eiskasten)_
```

Comment:
@Eiskasten is this really new in this release? Or was this just not removed from a previous version?

Reply:
This should be the 3rd highlight then. Was someone already able to fully run everything in the tutorial of the xfconf binding? We probably should merge this for the release, too.

Reply:
As I mentioned in #4948 it works for me now in the debian image.